external/storm-mqtt/pom.xml

```diff
@@ -73,7 +73,7 @@
         <dependency>
             <groupId>org.fusesource.mqtt-client</groupId>
             <artifactId>mqtt-client</artifactId>
-            <version>1.10</version>
+            <version>1.15</version>
```

Comment:
Also Update the DEPENDENCY and LICENSE files so that the build will go to success.